packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -364,6 +364,18 @@ class AccountInternal extends PureComponent {
       this.paged.unsubscribe();
     }
 
+    if (
+      query.state.orderExpressions &&
+      query.state.orderExpressions.length > 0
+    ) {
+      // If there are no order expressions in the query,
+      // transactions are sorted by sort_order by default.
+      // In case there are order expressions, let's add back
+      // the sort_order sorting so that transactions still
+      // maintain the correct order.
+      query = query.orderBy({ sort_order: 'desc' });
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should this logic also be added to mobile account component? And if yes: any way we could unify it so we don't have duplicate sorting logic?

(I don't mean actually implementing FULL drag-n-drop support on mobile; just that the list is ordered the same order on both desktop and mobile)